src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -810,11 +810,7 @@ export default class Pilot {
      * @method sayTargetedSpeed
      */
     sayTargetedSpeed() {
-        if (this._mcp.speed === MCP_MODE.SPEED.VNAV) {
-            // TODO: how do we handle the cases where there isn't a speedRestriction for a waypoint?
-            return [true, this._fms.currentWaypoint.speed];
-        }
-
+        // TODO: how do we handle the cases where there isn't a speed restriction for a waypoint?
```

Comment:
The question really should have been `// TODO: How do we handle the cases where aircraft are using VNAV speed?`. I didn't amend the previous TODO. done now.